src/test/kotlin/org/jetbrains/research/testspark/helper/ErrorMessageNormalizerTest.kt

```diff
@@ -0,0 +1,68 @@
+package org.jetbrains.research.testspark.display
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class ErrorMessageNormalizerTest {
+
+    /**
+     * Test with a string length less than BLOCK_SIZE and no separators.
+     */
+    @Test
```

Comment:
We also want to test the following condition in the implementation:
```kotlin
if (builder.endsWith(SEPARATOR) && (error.length % BLOCK_SIZE == 0)
```
There should be a test with a message whose size is multiple of `BLOCK_SIZE`, and the message should end with the `SEPARATOR`.